javagen/src/main/java/com/azure/autorest/Javagen.java

```diff
@@ -142,7 +142,12 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
             // Service version
             if (settings.isLowLevelClient() && settings.getServiceVersions() != null) {
                 String packageName = settings.getPackage();
-                String serviceName = settings.getServiceName().replaceAll("\\s", "");
+                String serviceName;
```

Comment:
do we need to add a test for the null case?

Reply:
Yes, we do.